versions.toml

```
@@ -1,4 +1,4 @@
 # Update these values when a new release is created.
 # `stacks-common/build.rs` will automatically update `versions.rs` with these values.
-stacks_node_version = "3.1.0.0.8"
-stacks_signer_version = "3.1.0.0.8.1"
+stacks_node_version = "3.1.0.0.9"
+stacks_signer_version = "3.1.0.0.9.1"
```

@wileyj I think this should be 3.1.0.0.9.0
```suggestion
stacks_signer_version = "3.1.0.0.9.0"
```